### DISCUSSION TITLE
get_all_tweets() returns 0 result

### DISCUSSION THREAD

**Discussion by binchen19** [cjbarrie/academictwitteR#246] [Q&A]
I've set up everything following the instruction. And I can see my tokens properly via get_bearer(). However, when I run a get_all_tweets(), it looks like the number of tweets I got is 0, and the size of the JSON file is only 2b. There are no warnings, no error messages. Does anyone know what's wrong with this?

**Comment by chainsawriot** [comment]
@binchen19 The most probable reason is no tweet matching your query. What did you search for? It would be super helpful if you can provide a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example).

**Reply by binchen19** [reply]
Thanks for your reply.  I figured that I can get some tweets if I manually input keywords in the `query ` of `get_all_tweets`, but if I set the query with `build_query` before (add some is: retweet limits), it returns 0 results.  I am not sure if I used the "build query" properly. Here are my codes:

```
query <- build_query(
          query = "#stopthesteal",
          is_retweet = TRUE,
          has_hashtags = TRUE,
          country = "US")

tweets <-
  get_all_tweets(
    query = query,
    start_tweets = "2020-11-03T00:00:00Z",
    end_tweets = "2020-11-04T00:00:00Z",
    data_path = "test/",
    n = 100000)
```
